src/mc_rbdyn/configuration_io.cpp

```diff
@@ -149,9 +149,31 @@ mc_rtc::Configuration ConfigurationLoader<mc_rbdyn::JointSensor>::save(const mc_
 
 mc_rbdyn::Collision ConfigurationLoader<mc_rbdyn::Collision>::load(const mc_rtc::Configuration & config)
 {
-  return mc_rbdyn::Collision(config("body1"), config("body2"), config("iDist", 0.05), config("sDist", 0.01),
-                             config("damping", 0.0), config("r1Joints", std::vector<std::string>{}),
-                             config("r2Joints", std::vector<std::string>{}));
+  auto body1 = config("body1");
+  auto body2 = config("body2");
+  auto loadDeprecatedActiveJoints = [&](std::string prefix)
+  {
+    auto r1Joints = prefix + "Joints";
+    auto r1ActiveJoints = prefix + "ActiveJoints";
+    auto activeJoints = std::vector<std::string>{};
+    if(config.has(r1Joints))
+    {
+      mc_rtc::log::deprecated(fmt::format("ConfigurationLoader<mc_rbdyn::Collision> (bodies: {} - {})", body1, body2),
+                              r1Joints, r1ActiveJoints);
+      if(auto r1ActiveJointsC = config.find(r1ActiveJoints))
+      {
+        mc_rtc::log::warning("ConfigurationLoader<mc_rbdyn::Collision> has both {0} and {1}, {0} will be ignored",
+                             r1Joints, r1ActiveJoints);
+        activeJoints = *r1ActiveJointsC;
+      }
+      else { activeJoints = config(r1Joints); }
+    }
+    return activeJoints;
```

Comment:
this is strange, you only search for `r1ActiveJoints` if there's already `r1Joints` in the configuration.

Given the other points I raised, I think:
```cpp
auto loadActiveJoints = [&](std::string prefix) -> std::tuple<std::optional<std::vector::string>>, bool>
{
	if(auto cfg = config.find(prefix + "ActiveJoints"))
	{
	  return {cfg.operator std::vector<std::string>(), false};
	}
        if(auto cfg = config.find(prefix + "InactiveJoints"))
	{
	  return {cfg.operator std::vector<std::string>(), true};
	}
	if(auto cfg = config.find(prefix + "Joints"))
	{
	  return {cfg.operator std::vector<std::string>(), false};
	}
	return {std::nullopt, false};
};
const auto & [r1Joints, r1Inactive] = loadActiveJoints("r1");
const auto & [r2Joints, r2Inactive] = loadActiveJoints("r2");
return mc_rbdyn::Collision(body1, body2, config("iDist", 0.05), config("sDist, 0.01), config("damping", 0.0), r1Joints, r2Joints, r1Inactive, r2Inactive);
```